src/main/java/com/exadel/discount/repository/CountryRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.exadel.discount.repository;
+
+import com.exadel.discount.entity.Country;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface CountryRepository extends JpaRepository<Country, UUID> {
+}
```

Comment:
Add here findAll(), findById, findByName visibly and write @EntityGraph(attributePaths = {"cities"}) over them. It helps to do just one query to DB instead of many, like now. Look at condole, hibernate shows all queries there

Reply:
> 
> 
> Add here findAll(), findById, findByName visibly and write @EntityGraph(attributePaths = {"cities"}) over them. It helps to do just one query to DB instead of many, like now. Look at condole, hibernate shows all queries there

Done